### ISSUE TITLE
Why Keras can use test data as validation data?

### ISSUE THREAD

**Issue by dsy412610** [fchollet/keras#2484]
I found that test data was using as validation in cifar10_cnn: fit(X_train, Y_train, batch_size=batch_size, nb_epoch=nb_epoch, validation_data=(X_test, Y_test), shuffle=True). Why Keras can use test data as validation data? Thanks!

**Comment by saicoco** [comment]
Just look test_data as valid_data, namely use the test_data evaluate the model at the end of epoch

**Comment by dsy412610** [comment]
Thanks for you quick reply! As far as I know, validation set is used to tune the parameters and test set should be some new images which are not contained in train set and validation set. But If I used test_data as valid_data, is that ok?

**Comment by saicoco** [comment]
cross-validation:we split the data to train_data and valid_data, use valid_data as monitor,we can save the modal by min val_loss or max val_acc,validation_data can not tune the parameters,when validation_data=(x_test, y_test), we can see it as x_train, x_test, y_train,y_test, = split_data(data, labels, test_size=percent)
